.github/workflows/cut-rc.yml

```
@@ -0,0 +1,503 @@
+  jira_projects: ROX, RS, RTOOLS
+  slack_channel: U02MPRJDANM
+
+jobs:
+  check-jira:
+    name: Check Jira tickets for release ${{inputs.version}}
+    runs-on: ubuntu-latest
+    env:
+      JIRA_TOKEN: ${{ secrets.JIRA_TOKEN }}
+    steps:
+      - name: Query JIRA
+        env:
+          JQL: >-
+            project IN (${{env.jira_projects}})
+            AND fixVersion = \"${{inputs.version}}\"
+            AND status != CLOSED
+            AND Component != Documentation AND type != Epic
+            order by assignee
+
+        run: |
+          GH_MD_FORMAT_LINE=$(cat << EOF
+          "* [\(.key)](https://issues.redhat.com/browse/\(.key)): **"+(.fields.assignee.displayName // "unassigned")+"** (\(.fields.status.name)) — _"+(.fields.summary | gsub (" +$";""))+"_"
+          EOF
+          )
+
+          ISSUES=$(curl -f -sSL --get --data-urlencode "jql=$JQL" \
+            -H "Authorization: Bearer $JIRA_TOKEN" \
+            -H "Accept: application/json" \
+            "https://issues.redhat.com/rest/api/2/search" \
+          | jq -r ".issues[] | $GH_MD_FORMAT_LINE" \
+          | sort)
+
+          if [ -n "$ISSUES" ]
+          then
+            cat << EOF >> $GITHUB_STEP_SUMMARY
+          The following Jira issues are still open for release ${{inputs.version}}:
+
+          $ISSUES
+
+          Please contact the assignees to clarify the status.
+          EOF
+            echo "::error::There are non-closed Jira issues for version ${{inputs.version}}."
+
+            curl -f -sSL --get --data-urlencode "jql=$JQL" \
+            -H "Authorization: Bearer $JIRA_TOKEN" \
+            -H "Accept: application/json" \
+            "https://issues.redhat.com/rest/api/2/search" \
+            | jq -r ".issues[] | .key" | while read -r KEY
+            do
+              curl -f -sSL -X POST \
+                -H "Authorization: Bearer $JIRA_TOKEN" \
+                --data '{"body": "Release ${{inputs.version}} is ongoing. Please update the status."}' \
+                -H "Content-Type: application/json" \
+                "https://issues.redhat.com/rest/api/2/issue/$KEY/comment"
+            done
+
+            exit 1
+          fi
+          echo "All issues for Jira release ${{inputs.version}} are closed." >> $GITHUB_STEP_SUMMARY
+
+  variables:
+    name: Setup variables
+    uses: stackrox/stackrox/.github/workflows/variables.yml@master
+    with:
+      version: ${{format('{0}{1}', github.event.milestone.title, github.event.inputs.version)}}
+
+  postpone-prs:
+    name: Postpone open PRs
+    needs: variables
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - name: Check open PRs
+        id: check
+        run: |
+          PRs=$(gh pr list -s open \
+          --repo "${{github.repository}}" \
+          --search "milestone:${{needs.variables.outputs.milestone}}" \
+          --json number \
+          --jq length)
+          if [ "$PRs" -gt 0 ]
+          then
```

I usually prefer the condensed version of the if:

```suggestion
          if [ "$PRs" -gt 0 ]; then
```